### PR TITLE
[beta-updates] veggieseasons

### PR DESCRIPTION
**Pull request by craiglabenz**
Updated `veggieseasons` for Dart 2.17

### CODE REVIEW

File: veggieseasons/lib/screens/details.dart [hunk 1]
```diff
@@ -255,10 +255,10 @@ class DetailsScreen extends StatefulWidget {
   }
 
   @override
-  _DetailsScreenState createState() => _DetailsScreenState();
+  DetailsScreenState createState() => DetailsScreenState();
```
Comment (domesticmouse) [inline veggieseasons/lib/screens/details.dart:258]: State<DetailsScreen> createState() => _DetailsScreenState();

**Review by domesticmouse — CHANGES_REQUESTED** [review]

File: veggieseasons/lib/screens/details.dart (hunk 1 above)
Comment (craiglabenz) [reply veggieseasons/lib/screens/details.dart]: Updated

File: veggieseasons/lib/screens/search.dart [hunk 2]
```diff
@@ -15,10 +15,10 @@ class SearchScreen extends StatefulWidget {
   final String? restorationId;
 
   @override
-  _SearchScreenState createState() => _SearchScreenState();
+  SearchScreenState createState() => SearchScreenState();
```
Comment (domesticmouse) [inline veggieseasons/lib/screens/search.dart:18]: Got another one =)

File: veggieseasons/lib/widgets/close_button.dart [hunk 3]
```diff
@@ -47,10 +47,10 @@ class ColorChangingIcon extends ImplicitlyAnimatedWidget {
   final double? size;
 
   @override
-  _ColorChangingIconState createState() => _ColorChangingIconState();
+  ColorChangingIconState createState() => ColorChangingIconState();
```
Comment (domesticmouse) [inline veggieseasons/lib/widgets/close_button.dart:50]: And another =)

File: veggieseasons/lib/widgets/trivia.dart [hunk 4]
```diff
@@ -13,7 +13,7 @@ class TriviaView extends StatefulWidget {
   const TriviaView({this.id, this.restorationId, Key? key}) : super(key: key);
 
   @override
-  _TriviaViewState createState() => _TriviaViewState();
+  TriviaViewState createState() => TriviaViewState();
```
Comment (domesticmouse) [inline veggieseasons/lib/widgets/trivia.dart:16]: Beep!

File: veggieseasons/lib/widgets/veggie_card.dart [hunk 5]
```diff
@@ -66,10 +66,10 @@ class PressableCard extends StatefulWidget {
   final Duration duration;
 
   @override
-  _PressableCardState createState() => _PressableCardState();
+  PressableCardState createState() => PressableCardState();
```
Comment (domesticmouse) [inline veggieseasons/lib/widgets/veggie_card.dart:69]: Beep beep!

Comment (craiglabenz) [reply veggieseasons/lib/widgets/veggie_card.dart]: Okay, looked for them all this time

**Pull request merged**